src/libs/actions/NetworkRequestQueueUtils.js

```diff
@@ -0,0 +1,74 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as OptionsListUtils from '../OptionsListUtils';
+
+let persistedRequests;
+Onyx.connect({
+    key: ONYXKEYS.NETWORK_REQUEST_QUEUE,
+    callback: (val) => {
+        if (!val) {
+            return;
+        }
+        persistedRequests = val;
+    },
+});
+
+/**
+ * Modifies the persisted request of Policy_Employees_Merge API
+ * If removing a user after inviting them in offline mode, this function will modify the Request_Policy_Employees_Merge
+ * request to exclude the removed email(s)
+ *
+ * @param {String} policyID
+ * @param {Array<String>} logins emails to be exluded
+ */
+function modifyPersistedRequest_Policy_Employees_Merge(policyID, logins) {
```

Comment:
I think when this is moved to policy.js action. then we can rename it. 
```suggestion
function modifyInvitePersistedRequests(policyID, logins) {
```

Reply:
Like above, will remove this file.